pyxem/utils/io_utils.py

```diff
@@ -328,24 +345,31 @@ def _parse_hdr(fp):
 
 
 def _add_crosses(a):
-    """ Adds 3 pixel buffer cross to quad chip data.
+    """ Adds 3 pixel buffer cross to quad chip data. the input can be a stack or a reshaped 4DSTEM dask object. It returns
+    the object with the same number of axes.
 
     Parameters
     ----------
     a : dask.array
-        Stack of raw frames, prior to dimension reshaping, to insert
+        Stack of raw frames or reshaped dask array object, prior to dimension reshaping, to insert
         3 pixel buffer cross into.
 
     Returns
     -------
     b : dask.array
-        Stack of frames including 3 pixel buffer cross.
+        Stack of frames or reshaped 4DSTEM object including 3 pixel buffer cross in the diffraction plane.
     """
     # Determine dimensions of raw frame data
     a_type = a.dtype
+    original_shape = a.shape
     a_shape = a.shape
```

Comment:
Why do you need two different variables (original_shape and a_shape) defined identically? For code clarity I'd prefer to not overwrite a_shape below.